apps/hip/models.py

```diff
@@ -163,7 +163,7 @@ def updated_date(self):
 
 class QuickLinkCard(blocks.StructBlock):
     title = blocks.CharBlock(
-        max_length=255,
+        max_length=80,
         required=True,
         help_text=("The linked text that will be visible to the reader"),
```

Comment:
Let's update the `help_text` to add "(maximum of 80 characters)"

Reply:
done